modules/web/js/ballerina/ast/service-definition.js

```diff
@@ -319,6 +319,19 @@ define(['lodash', './node', 'log', '../utils/common-utils'],
         return connectorReference;
     };
 
+    /**
+     * Get all the connector references in the immediate scope
+     * @return {Array} connectorReferences
+     */
+    ServiceDefinition.prototype.getConnectorsInImmediateScope = function () {
```

Comment:
can we make this a util method in some util module? since it's been repeated a few times? Maybe we can move it to factory. so it will take a node as a arg and return connector refs within it's immediate children?